make deps fails because of sqlite3 deps

# Required information

 * Distribution: Ubuntu
 * Distribution version: 20.04

# Issue description

When trying to build from source despite following the install from source steps first `make deps` fails with `No package 'sqlite3' found`

```
daniel@test1:~/go/src/github.com/lxc/lxd$ make deps
checking for SQLITE... no
configure: error: Package requirements (sqlite3 >= 3.22.0) were not met:

No package 'sqlite3' found

Consider adjusting the PKG_CONFIG_PATH environment variable if you
installed software in a non-standard prefix.

Alternatively, you may set the environment variables SQLITE_CFLAGS
and SQLITE_LIBS to avoid the need to call pkg-config.
See the pkg-config man page for more details.
make: *** [Makefile:48: deps] Error 1
``` 

I was able to get this working by running `apt-get install libsqlite3-dev`

# Steps to reproduce

 1. follow build from source steps 
 2. run `make deps`

--- a/doc/index.md
+++ b/doc/index.md
@@ -42,7 +42,7 @@ later to work. On ubuntu, you can get those with:
 
 ```bash
 sudo apt update
-sudo apt install acl autoconf dnsmasq-base git golang libacl1-dev libcap-dev liblxc1 liblxc-dev libtool libudev-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables
+sudo apt install acl autoconf dnsmasq-base git golang libacl1-dev libcap-dev liblxc1 liblxc-dev libsqlite3-dev libtool libudev-dev libuv1-dev make pkg-config rsync squashfs-tools tar tcl xz-utils ebtables
 ```
 
 Note that when building LXC yourself, ensure to build it with the appropriate

